src/app/core/entity-components/entity-list/entity-list.component.ts

```diff
@@ -259,9 +275,14 @@ export class EntityListComponent<T extends Entity>
         ));
       case "prebuilt":
         return (filter.options = (config as PrebuiltFilterConfig<T>).options);
+      case "configurable-enum":
+        return (filter.options = this.createConfigurableEnumFilterOptions(
+          config as ConfigurableEnumFilterConfig<T>
+        ));
       default: {
         const options = [...new Set(this.entityList.map((c) => c[config.id]))];
         filter.initOptions(options, config.id);
+        return filter.options;
```

Comment:
I think I made a mistake concerning the return type here. The other options only return `null`